client/verta/docs/tutorials/packaging/custom_model.rst

```diff
@@ -0,0 +1,58 @@
+Creating custom models
```

Comment:
I'd like to use title case for the page titles, if we can (at least anything longer than three letters).
```suggestion
Creating Custom Models
```